### ISSUE TITLE
views should be findable by their accessibilityIdentifier to help support Localization

### ISSUE THREAD

**Issue by jmoody** [calabash/calabash-ios#20]
I have a problem similar to this (closed) [issue 13](https://github.com/calabash/calabash-ios/issues/13).

I have some views that I have only been able to find by setting their accessibilityLabel to a string that is not appropriate for Localization.

iOS 5 introduced the  **accessibilityIdentifier** so I think that it makes sense for calabash support finding views by this property.
